R/SS_output.R

```diff
@@ -1883,6 +1883,12 @@ SS_output <-
           recruit_dist_endyr <- match_report_table("RECRUITMENT_DIST_endyr", 1,
             header = TRUE, type.convert = TRUE
           )
+          # temp fix  for 3.30.19
```

Comment:
I did already modify it in Stock Synthesis (in commit https://github.com/nmfs-stock-synthesis/stock-synthesis/commit/d618c3c0c6d58da9f532ed531149e1186b31b4e3), but this will still be needed for .19 models. I shouldn't have labeled this a "temp fix", I think it is a fine permanent fix for the missing space in .19 Report files.

Reply:
That makes complete sense now. Thanks for the explanation. Maybe we should note the SS3 commit to fix the missing space in the comment instead of temp fix.